src/status_im/ui/components/tabbar/styles.cljs

```diff
@@ -83,6 +83,11 @@
   {:height     tabs-height
    :align-self :stretch
    :ios        {:background-color :white
+                :shadow-radius    4
```

Comment:
why this?

Reply:
There is no change in ios style here, I just added a copy of this style for desktop (if I correctly understood your question). Changes highlighted incorrectly in github.